packages/cfd/src/Containers/__tests__/compare-accounts-content.spec.js

```diff
@@ -0,0 +1,202 @@
+        render(<CompareAccountsContent {...mock_props} landing_companies={all_landing_companies} />);
+        expect(screen.getByText('Synthetic')).toBeInTheDocument();
+        expect(screen.getByText('CFDs')).toBeInTheDocument();
+        expect(screen.getByText('Financial STP')).toBeInTheDocument();
+    });
+
+    it('should render the proper data', () => {
+        render(<CompareAccountsContent {...mock_props} landing_companies={all_landing_companies} />);
+        expect(screen.getByText(/Account Currency/i)).toBeInTheDocument();
+        expect(screen.getByText('EUR')).toBeInTheDocument();
+        expect(screen.getByText('EUR/GBP/USD')).toBeInTheDocument();
+        expect(screen.getByText('USD')).toBeInTheDocument();
+
+        expect(screen.getAllByText(/Maximum leverage/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/1/i)[0]).toBeInTheDocument();
+        expect(screen.getByText(/Up to 1:1000/i)).toBeInTheDocument();
+        expect(screen.getByText(/Up to 1:30/i)).toBeInTheDocument();
+        expect(screen.getAllByText(/Up to 1:100/i)[0]).toBeInTheDocument();
+
+        expect(screen.getAllByText(/Order execution/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/2/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/Market/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/Market/i)[1]).toBeInTheDocument();
+        expect(screen.getAllByText(/Market/i)[2]).toBeInTheDocument();
+
+        expect(screen.getAllByText(/Spread/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText('Fixed/Variable')[0]).toBeInTheDocument();
+        expect(screen.getAllByText('Fixed/Variable')[1]).toBeInTheDocument();
+        expect(screen.getAllByText('Variable')[0]).toBeInTheDocument();
+
+        expect(screen.getAllByText(/Commission/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/4/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/No/i)[1]).toBeInTheDocument();
+        expect(screen.getAllByText(/No/i)[2]).toBeInTheDocument();
+        expect(screen.getAllByText(/No/i)[3]).toBeInTheDocument();
+
+        expect(screen.getAllByText(/Minimum deposit/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/No/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/No/i)[1]).toBeInTheDocument();
+        expect(screen.getAllByText(/No/i)[2]).toBeInTheDocument();
+
+        expect(screen.getAllByText(/Margin call/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/5/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/100%/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/100%/i)[1]).toBeInTheDocument();
+        expect(screen.getAllByText(/100%/i)[2]).toBeInTheDocument();
+
+        expect(screen.getAllByText(/Stop out level/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/6/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText(/50%/i)[0]).toBeInTheDocument();
```

Comment:
I think we need not have case insensitive flag for numbers - eg: Line 73, 84, 85